libsycl/include/sycl/platform.hpp

```diff
@@ -0,0 +1,24 @@
+//==---------------- platform.hpp - SYCL platform --------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#pragma once
+
+#include <sycl/detail/export.hpp>
+
+namespace sycl {
+inline namespace _V1 {
```

Comment:
I don't think we've proven to ourselves that this helps us in any meaningful way.

Reply:
Using a versioned inline namespace is consistent with what libc++ does. Is it good to do likewise? I'm not sure. Perhaps worth getting an opinion from libc++ maintainers.